index.html

```diff
@@ -60,15 +60,90 @@ <h3 class="headline-text-1">Alick Nyirenda</h3>
       </div>
     </div>
     <div class="dividing-line"></div>
-
     <ul class="social">
         <li><a href="#"><i class="social-icon fa fa-github"></i></a></li>
         <li><a href="#"><i class="social-icon fa fa-linkedin"></i></a></li>
         <li><a href="#"><i class="social-icon fa fa-angellist clr-secondary"></i></a></li>
         <li><a href="#"><i class="social-icon fa fa-twitter"></i></a></li>
         <li><a href="#"><i class="social-icon fa fa-medium"></i></a></li>
     </ul>
+    <div class="work-arrow-container">
+      <span class="work-arrow"></span>
+      <p class="work-arrow-text">Work</p>
+    </div>
+  </section>
+
```

Comment:
- [ ] You have a horizontal overflow. Kindly fix it.
- [ ] The social media icons are a lot smaller than the ones in the design template. Kindly fix it.
_See the images below for clarity._

**Original** | **Your clone**
  ----------- | -----------
  ![image](https://user-images.githubusercontent.com/49621279/121549394-ad1b0680-c9c2-11eb-8e83-6a8e758c886b.png) | ![image](https://user-images.githubusercontent.com/49621279/121549446-b86e3200-c9c2-11eb-85ef-d493e013ad0a.png)

Reply:
Well resolved the issue!